packages/grid/src/features/standard-table/components/StandardTableHeadItem.tsx

```diff
@@ -86,6 +86,7 @@ export const StandardTableHeadItem = React.memo(
             : stickyProps.sticky
             ? "var(--swui-sticky-group-header-z-index)"
             : zIndex) as CSSProperties["zIndex"],
+          height: 0, // CSS trick to force TableHeadItem to use 100% height
```

Comment:
Not a perfect solution but it solves the problem of the current non-expanding header. 

Alternative solution could be passing an optional header height property to the StandardTable config?